app/src/main/java/com/framgia/awesomecity/screen/customer/booking/BookingViewModel.java

```
@@ -26,4 +46,78 @@ public void onStop() {
     public void setPresenter(BookingContract.Presenter presenter) {
         mPresenter = presenter;
     }
+
+    @Bindable
+    public String getCapacity() {
+        return mCapacity;
+    }
+
+    public void setCapacity(String capacity) {
+        mCapacity = capacity;
+        notifyPropertyChanged(BR.capacity);
+    }
+
+    @Bindable
+    public String getDate() {
+        return mDate;
+    }
+
+    public void setDate(String date) {
+        mDate = date;
+        notifyPropertyChanged(BR.date);
+    }
+
+    @Bindable
+    public String getTime() {
+        return mTime;
+    }
+
+    public void setTime(String time) {
+        mTime = time;
+        notifyPropertyChanged(BR.time);
+    }
+
+    public void onButtonClicked(View button) {
+
+    }
+
+    public void showDatePicker(View view) {
+        Calendar calendar = Calendar.getInstance();
+        DatePickerDialog pickerDialog = new DatePickerDialog(mContext,
+                new DatePickerDialog.OnDateSetListener() {
+                    @Override
+                    public void onDateSet(DatePicker datePicker, int year, int month, int day) {
+                        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
```

WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.